src/ontology/reports/sync-synonym/ordo.synonyms.confirmed.robot.tsv

```diff
@@ -1551,7 +1551,7 @@ MONDO:0008038	ataxia-pancytopenia syndrome	oio:hasExactSynonym		myelocerebellar
 MONDO:0008039	tropical spastic paraparesis	oio:hasExactSynonym		HTLV-1-associated myelopathy/tropical spastic paraparesis			Orphanet:289326	Tropical spastic paraparesis					HTLV-1-associated myelopathy/tropical spastic paraparesis	Orphanet:289326										
 MONDO:0008039	tropical spastic paraparesis	oio:hasExactSynonym		Human T-lymphotropic virus type I-associated myelopathy/tropical spastic paraparesis			Orphanet:289326	Tropical spastic paraparesis					Human T-lymphotropic virus type I-associated myelopathy/tropical spastic paraparesis	Orphanet:289326										
 MONDO:0008039	tropical spastic paraparesis	oio:hasExactSynonym		Human T-lymphotropic virus type-1-associated myelopathy/tropical spastic paraparesis			Orphanet:289326	Tropical spastic paraparesis					Human T-lymphotropic virus type-1-associated myelopathy/tropical spastic paraparesis	Orphanet:289326										
-MONDO:0008039	tropical spastic paraparesis	oio:hasExactSynonym		ham/TSP	ham/TSP	HAM/TSP	Orphanet:289326	Tropical spastic paraparesis	http://purl.obolibrary.org/obo/mondo#ABBREVIATION				ham/TSP	Orphanet:289326	http://purl.obolibrary.org/obo/mondo#ABBREVIATION									
+MONDO:0008039	tropical spastic paraparesis	oio:hasExactSynonym		HAM/TSP	ham/TSP	HAM/TSP	Orphanet:289326	Tropical spastic paraparesis	http://purl.obolibrary.org/obo/mondo#ABBREVIATION				HAM/TSP	Orphanet:289326	http://purl.obolibrary.org/obo/mondo#ABBREVIATION									
```

Comment:
## Synonym Sync diffs
As observed in other recent PRs and I mentioned on Slack in `mondo-ingest` today (don't have the link), many/most of these are due to:
- #684 

I say that because the n `+` and `-` are about the same, and when I look through the diffs I see mostly these cases.

However I do see a lot of other cases, at least in this one Orphanet file, such as highlighted here, where the capitalization has changed recently, so some of the diffs are attributable to that.